javascript/javascript2/week2/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+<body>
+    <ul id="searchResult">
+        hi
+    </ul>
+    <div id="result"></div>
+    <script src="warmUp.js"></script>
+    <script src="classExercises.js"></script>
+    <script src="moviesEx.js"></script>
+    <script src="moviesArray.js"></script>
```

Comment:
You have not committed `moviesArray.js` to this branch.

Reply:
yes , i removed this page . can you help me , how can i use movies array from moviesArray.js file in main.js file ?

Reply:
If you include the file like this, are the movies available to console.log(movies)?

```suggestion
    <script src="moviesArray.js"></script>
    <script src="moviesEx.js"></script>
```